samples/XCT.Sample/Pages/TestCases/TouchEffectCollectionViewPage.xaml.cs

```diff
@@ -1,19 +1,16 @@
-﻿
-using System.Windows.Input;
-using Xamarin.Forms;
+﻿using System.Windows.Input;
+using Xamarin.CommunityToolkit.ObjectModel;
 
 namespace Xamarin.CommunityToolkit.Sample.Pages.TestCases
 {
 	public partial class TouchEffectCollectionViewPage
 	{
-		ICommand longPressCommand;
-
 		public TouchEffectCollectionViewPage()
-			=> InitializeComponent();
-
-		public ICommand LongPressCommand => longPressCommand ??= new Command(() =>
 		{
-			this.DisplayAlert("Long Press", null, "OK");
-		});
+			InitializeComponent();
+			LongPressCommand = new AsyncCommand(() => DisplayAlert("Long Press", null, "OK"));
```

Comment:
nit: Shouldn't `CommandFactory` be used here? 
I remember that we planned to discuss it's fate latter, but for now it's used everywhere inside sample app. And this is exact situation it was created to prevent (`Command` was incorrectly used here instead of `AsyncCommand`)